specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/examples/UsageDetailsExpand.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "api-version": "2018-01-31",
+    "scope": "subscriptions/subid/providers/Microsoft.Billing/billingPeriods/201702",
+    "$expand": "meterDetails,additionalProperties",
+    "$filter": "usageEnd le 2017-02-14T00:00:00Z",
+    "$top": "1"
```

Comment:
@bgsky Change this to integer instead of a string

Reply:
Got it

Reply:
Please change it here